bin/n

```diff
@@ -4,13 +4,29 @@
 # Setup.
 #
 
-VERSION="2.0.2"
+VERSION="1.3.0"
 UP=$'\033[A'
 DOWN=$'\033[B'
 N_PREFIX=${N_PREFIX-/usr/local}
 BASE_VERSIONS_DIR=$N_PREFIX/n/versions
 
 #
+# Log <type> <msg>
+#
+
+log() {
+  printf "  \033[36m%10s\033[0m : \033[90m%s\033[0m\n" $1 $2
+}
+
+#
+# Exit with the given <msg ...>
+#
+
+abort() {
+  printf "\n  \033[31mError: $@\033[0m\n\n" && exit 1
+}
```

Comment:
ditto, so it's weird diff, I didn't track the commits history of @wblankenship, just guess his master branch might not be updated from `tj/n:master` :)